### PR TITLE
Minor content changes on pricing page

### PR DESCRIPTION
**Pull request by MichaelAring**
## Description

- Closes [#3168](https://github.com/gitpod-io/website/issues/3168)

### CODE REVIEW

File: src/routes/pricing.svelte [hunk 1]
```diff
@@ -30,10 +30,10 @@
 <Section>
   <div class="mb-small md:mb-x-medium">
     <h2 id="cost-estimator" class="text-center h2 mb-macro md:!mb-micro">
-      Cost estimator
+      Pricing calculator
```
[the remark this answers is not in the document]
Comment (Cahllagerfeld) [reply src/routes/pricing.svelte:33]: I saw the change being deployed in the dashboard already, so making a change to the ID would require a new deployment of the dashboard

File: src/lib/components/pricing/plans-and-pricing.svelte [hunk 2]
```diff
@@ -22,7 +22,7 @@
     {#if pricingPlans.length > 0}
       <PricingBoxes {pricingPlans} />
       <p class="mt-micro text-center">
-        All prices excluding VAT. See complete <a
+        All prices excluding VAT. 1 credit = $0.036. See complete <a
```
Comment (Siddhant-K-code) [inline src/lib/components/pricing/plans-and-pricing.svelte:25]: we need to add the same functionality like #3170 here.

Comment (Cahllagerfeld) [reply src/lib/components/pricing/plans-and-pricing.svelte:25]: could you add it real quick? 😬

File: src/routes/pricing.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (svenefftinge) [reply src/routes/pricing.svelte:33]: yes, let's keep the ID.

**Pull request merged**